packages/flutter_tools/test/general.shard/windows/visual_studio_test.dart

```diff
@@ -306,6 +362,16 @@ void main() {
         fixture.fileSystem,
         fixture.processManager,
       );
+      setMockCompatibleVisualStudioBuildToolsInstallation(
+        null,
+        fixture.fileSystem,
+        fixture.processManager,
+      );
+      setMockPrereleaseVisualStudioBuildToolsInstallation(
+        null,
+        fixture.fileSystem,
+        fixture.processManager,
+      );
```

Comment:
This block of four is used so often it would be good to just make a helper at this point called something like `setNoViableToolchainInstallation` that does all four.